app/src/main/java/com/main/mealplanner/MainViewModel.kt

```diff
@@ -10,10 +10,11 @@ class MainViewModel : ViewModel() {
     var recipes: MutableLiveData<ArrayList<RecipeHeader>> = MutableLiveData()
     var recipeService: RecipeService = RecipeService()
 
-    fun fetchRecipe(s: String) {
-        recipes = recipeService.fetchRecipe(s)
+    fun fetchRecipe(meal: String) {
+        recipes = recipeService.fetchRecipe(meal)
```

Comment:
using s as a name is an awful habit to get into, changed it to meal so you can easily tell what the contents of the string probably is

Reply:
Agreed.  Descriptive variables names == readable code == less need for comments.